### ISSUE TITLE
"Most recent version" seems to be way out of date. 

### ISSUE THREAD

**Issue by vita10gy** [BurntSushi/nfldb#282]
Am I missing something or has the DB available at 

>The most recent version of the nfldb PostgreSQL database is available here: http://burntsushi.net/stuff/nfldb/nfldb.sql.zip.

not updated in forever? 

I kind of just need the DB, ideally bypassing a python based install. This worked for a while and then just stopped.

**Comment by BurntSushi** [comment]
You are correct that it hasn't been updated in quite some time. I don't have any immediate plans to update it. You'll need to do it yourself.

**Comment by vita10gy** [comment]
Bummer. Thanks for the system in general though. You helped me turn S.Bradford into Sam Bradford and otherwise sort out the 22 T.Willams's that play for every team on my website.
